guides-markdown/vps-setup.md

```diff
@@ -0,0 +1,129 @@
+
+6. Choose a data center region.
+7. Add a new ssh key
+
+### Setting Up SSH
+If you already have an ssh key you can copy your pubkey here. If you do not then follow these steps:
+
+`ssh-keygen -t rsa -C "your_email@example.com"`
+
+Accept the defaults.
+
+On Windows your keys are located:
+
+`cd %userprofile%/.ssh`
+
+On Linux/Mac:
+
+`cd ~/.ssh`
+
+Copy the contents of the `id_rsa.pub` file and paste them into Digital Ocean's form.
+
+`ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCsYKEA5LZCDyMF+ZbrPWeVIYso0ZzpIZx9L7R+CGqMPo0mrSlYeeaPbP1btM/Wis4a81EaTM7Y5kkKqZ4XB/LnRWp415XVl5QdtGF2l5tgiy2ootVxEwdrH0lXyGFHEpOwHU6MYdYCd+bgpQwa291Q4bOUJhGxNZ07L/rMtZfWhWL
++YL+JpSajg/uonu+4YKuFETggGLIuK+piTD9dvjiaThwKtqiCh2dnqdHztRYk+OehJUcof3tFl9kSRUmh9MVI7pDaOxCJWRaU1dsn9YaUwRkIyOwESHqBdCE9ZDU4FzNItRh2dYY4ukGv2iRqZoTrjcB8UGJepI65aINKNvdj email@nomail.com`
+
+8. Choose a hostname for the droplet and click create.
+
+Digital Ocean will provision your server. Now is a good time to grab coffee.
+
+
+## Amazon AWS
+
+1. Create an account at https://aws.amazon.com
+2. Launch a new instance from the console https://aws.amazon.com/console
+3. Choose 'Ubuntu 16.04' and at least 2CPU 4GiB for hardware.
+4. Continue to add storage. Add 500GiB.
+
+### A Note on hardware requirements
+These hardware requirements are for a full node. When run in SPV node, you will not need the additional block storage. To run a full node you must maintain a complete history of all Bitcoin transactions. Without the full history the node is unable to validate transactions. If it cannot validate transactions, it cannot validate blocks or mine transactions to blocks.
+
+SPV Nodes use something called a bloom filter to maintain a smaller set of records. For example, if your Bitcoin address were `1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa` your SPV node will ask its network peers for all transactions with an address that starts with `1A1z`. The returned data set will contain extra transactions irrelevant to your wallet. But its storage requirements will be far smaller than a full node. Consequently, SPV Nodes can only verify their own transactions.
+
+5. Review and Launch
+6. Amazon will prompt you for an ssh keypair, download a new keypair.
+7. Launch Instances.
+8. View Instances
+
+Amazon will provision your server. Now is a good time to grab coffee.
+
+
+## Connecting to your server
+
+#### Digital Ocean
+
+`ssh Your.Server.Ip.Here`
+
+E.g.
+
+`ssh 178.62.124.90`
+
+Accept the RSA key, and you will be at the command line
+
+
+#### Amazon
+
+In the same directory as the private key you downloaded,
+
+```chmod 400 test.pem
```

Comment:
when doing code blocks, the text immediately after the triple back tick (and on the same line) is interpreted as the language when converting to html (we use `marked.js` for conversions). so in this case the `chmod` line should be on the next line, and you could specify `bash` as the language after the back ticks